syntax = "proto3";

package hello;


message HelloRequest {
  string name = 1;
}

message HelloResponse {
  string response = 2;
}

service Hello {
  rpc SayHello(HelloRequest) returns (HelloResponse) {}
}
